dropshot/src/lib.rs

```diff
@@ -565,16 +568,16 @@ pub(crate) struct ResponseInfo {
 #[usdt::provider]
 mod dropshot {
     use crate::{RequestInfo, ResponseInfo};
-    fn request_start(_: RequestInfo) {}
-    fn request_finish(_: ResponseInfo) {}
+    fn request_start(_: &RequestInfo) {}
+    fn request_finish(_: &ResponseInfo) {}
 }
 
+/// Register DTrace probes exposed by Dropshot.
+///
+/// This method _must_ be called for any of the DTrace probes in Dropshot
+/// to be visible from the `dtrace(1M)` command-line tool.
 pub fn register_probes() -> Result<(), usdt::Error> {
-    if cfg!(feature = "usdt-probes") {
-        usdt::register_probes()
-    } else {
-        Ok(())
-    }
+    usdt::register_probes()
```

Comment:
Is this no longer gated behind a feature?  I'm probably misremembering but I thought this was to keep things working on "stable".

Reply:
There's still a feature flag for Dropshot. This function is always available in `usdt`, regardless of whether a real or no-op implementation is selected, to keep the public API the same in any case.